### DISCUSSION TITLE
Issue with importing RainbowKitChainContext module from @rainbow-me/rainbowkit package

### DISCUSSION THREAD

**Discussion by ghost** [rainbow-me/rainbowkit#1211] [Feedback]
I am writing to seek assistance with an issue that I am facing while trying to import the RainbowKitChainContext module from the @rainbow-me/rainbowkit package.

I have followed all the steps mentioned in the documentation and have installed the package correctly. However, when I try to import the RainbowKitChainContext module using the following syntax:

import { RainbowKitChainContext } from '@rainbow-me/rainbowkit';
Copy
I get an error message that says "Missing "./dist/components/RainbowKitProvider/RainbowKitChainContext" specifier in "@rainbow-me/rainbowkit" package". I have tried various solutions such as reinstalling the package and clearing my cache, but none of them seem to work.

Can you please help me resolve this issue? Do let me know if there is any additional information that I can provide to help diagnose this problem.

the issue 
"Internal server error: Missing "./dist/components/RainbowKitProvider/RainbowKitChainContext" specifier in "@rainbow-me/rainbowkit" package"

**Comment by DanielSinclair** [comment]
If you're still seeing this issue, please open a new Issue on the repo and share a reproducible example.
